src/libslic3r/Extruder.cpp

```diff
@@ -94,6 +94,10 @@ double Extruder::unretract()
     }
 }
 
+double Extruder::need_unretract() {
+    return m_retracted + m_restart_extra != 0;
```

Comment:
Looks like this function is not used anymore?
If we intended to keep this function, then it would be better to keep handle m_share_extruder condition too?

Reply:
I'll scrap the function - it was used when the algorithm was exploring extended travel beyond 50% nozzle size but I've seen in Superslicer that this resulted in the deretraction being done on neighbouring external surfaces if only 2 perimeters were present (like the benchy flag pole). So decided to play it safe and not implement this. I'll remove the functions.

Reply:
Done.